config/routes.rb

```diff
@@ -15,6 +15,7 @@
       resources :tasks, except: [:index] do
         member do
           put :move
+          delete :delete_file_attachment
```

Comment:
doesn't it look/sound weird? 
mb `remove_attachment`?

Reply:
remove_attachment is good )